projects/rtmpose/rtmpose/face_2d_keypoint/rtmpose-m_8xb64-120e_lapa-256x256.py

```diff
@@ -186,8 +180,8 @@
         type=dataset_type,
         data_root=data_root,
         data_mode=data_mode,
-        ann_file='annotations/lapa_train.json',
-        data_prefix=dict(img='train/images/'),
+        ann_file='annotations/lapa_trainval.json',
+        data_prefix=dict(img=''),
```

Comment:
Since the file structure of the lapa dataset has changed, please update the introduction to this dataset in the dataset_zoo documentation accordingly